start/scheduler.ts

```diff
@@ -0,0 +1,11 @@
+import scheduler from 'adonisjs-scheduler/services/main'
+import { runScrapper } from '../scripts/scrapper.js'
+import logger from '@adonisjs/core/services/logger'
+
+scheduler
+  .call(() => {
+    logger.info('Running scraper.')
+    runScrapper().then()
```

Comment:
what's this `.then()` here for? does it have a purpose?

Reply:
<img width="624" alt="image" src="https://github.com/user-attachments/assets/650df3a1-abe6-4d18-b995-b5e390796fcf">

i just really don't like warnings, thats all

Reply:
try to write `void runScraper()` instead? not sure if this will help

Reply:
yeah it works, thanks from my OCD